pkg/infra/pulumi_aws/iac/load_balancing.ts

```diff
@@ -32,10 +23,13 @@ export class LoadBalancerPlugin {
         params: LoadBalancerArgs
     ): aws.lb.LoadBalancer => {
         let lb: aws.lb.LoadBalancer
+        let lbName = sanitized(validators.loadBalancer.nameValidation())`${h(appName)}-${h(
+            resourceId
+        )}`
         switch (params.loadBalancerType) {
             case 'application':
                 lb = new aws.lb.LoadBalancer(`${appName}-${resourceId}-alb`, {
-                    name: `${appName}-${resourceId}`,
+                    name: lbName,
```

Comment:
also a general thing, we likely shouldnt set actual names of resources because then they cant get replaced if they change. We should be doing the validation on the resource id for pulumi i believe so that it autogenerates the name + some uuid for uniqueness if we ever need replacements.

let me know your thought on that, i think i added the name field, but i was going to remove it in my next pr because i noticed it could cause issues. We should likely see where else we do this